### DISCUSSION TITLE
Permission denied. Could not authenticate the credentials.

### DISCUSSION THREAD

**Discussion by mAghaDash** [Laravel-Backpack/community-forum#984] [Q&A (Help)]
I'm trying to install some pro packages using the command:

`php artisan backpack:require:editablecolumns
`
However, I received a "Permission denied. Could not authenticate the credentials" error. I was prompted to enter a token, and I tried using my username and password from https://backpackforlaravel.com/user/tokens as a token, but I keep encountering the same error. I do have a backpack/pro subscription.
how can I fix this?
what is exactly the token and how I can enter it?

**Comment by karandatwani92** [comment]
Hey @mAghaDash 

1. Can you check the `Updates Until` column on [https://backpackforlaravel.com/user/tokens](https://backpackforlaravel.com/user/tokens) page? This tells if you have **access** to the latest version or not.
1. Can you try the [manual installation](https://backpackforlaravel.com/products/editable-columns). 
```bash
composer require backpack/editable-columns
```

If you continue having the issue, please let me know.
